src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Tree/Visitors/ExpressionTreeRewriter.cs

```diff
@@ -104,7 +104,7 @@ protected override Expr VisitBOUNDLAMBDA(ExprBoundLambda anonmeth)
 
             CType delegateType = anonmeth.DelegateType;
             TypeArray lambdaTypeParams = GetSymbolLoader().getBSymmgr().AllocParams(1, new CType[] { delegateType });
-            AggregateType expressionType = GetSymbolLoader().GetOptPredefTypeErr(PredefinedType.PT_EXPRESSION);
+            AggregateType expressionType = GetSymbolLoader().GetPredefindType(PredefinedType.PT_EXPRESSION);
```

Comment:
Made me wonder for a moment - "Why are we not handling the error? What if the runtime does not have the type?"
But we statically know that we have the type. Because of the eventual `typeof` there is no such failure mode to handle as "type is missing".